src/PKSim.Presentation/Presenters/Charts/IndividualSimulationComparisonPresenter.cs

```diff
@@ -122,7 +120,7 @@ private void showChartView()
 
       protected override string NameForColumn(DataColumn dataColumn)
       {
-         return _chartPresenterContext.QuantityDisplayPathMapper.DisplayPathAsStringFor(SimulationFor(dataColumn), dataColumn, addSimulationName: true);
+         return _chartPresenterContext.CurveNamer.CurveNameForColumn(SimulationFor(dataColumn), dataColumn);
```

Comment:
I am going to merge that but this method should probably be defined in ChartPresenterContext as well (that would just delegate to the CurveNamer)